plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/conf/hydra/sweeper/nevergrad-sweeper.yaml

```diff
@@ -0,0 +1,11 @@
+hydra:
+  sweeper:
+    class: hydra_plugins.hydra_nevergrad_sweeper.NevergradSweeper
+    params:
+      optimizer: OnePlusOne
+      num_workers: 10
+      budget: 80
+      noisy: false
+      maximize: false
+      seed: null
+      version: 1
```

Comment:
1. Can you add comments documenting those options?
2. Can you add a link to the docs as well?